docs/learning/ui/core/Settings.md

```diff
@@ -0,0 +1,150 @@
+# Settings
+
+The [SettingsManager]($ui-core) allows the registration of [SettingsProvider]($ui-core) classes to provide the settings to display with the [SettingsContainer]($ui-core) component. In an application that employees App-UI, the [SettingsModalFrontstage]($ui-framework) frontstage will display the SettingsContainer and its entries.
+
+## SettingsTabEntry
+
+Registered [SettingsProvider]($ui-core) instance will implement the method `getSettingEntries` to return an array of [SettingsProvider]($ui-core) items. Each SettingsTabEntry will populate a Tab entry in the [SettingsContainer]($ui-core) component. The `tabId` property is a string and must be unique across all registered SettingsProviders. A common practice is to prefix the tabId with the package name to ensure uniqueness. The `page` property holds the React.Element that will be used to construct the component to edit settings for this entry.  It is the `page's` responsibility to persist and retrieve persisted settings. If the `page` contains multiple properties that must be saved together the SettingEntry can set the `pageWillHandleCloseRequest` property to `true`. The `page's` control should then register to be notified when the setting container is closing or when the active SettingsEnty is changing so that any unsaved data can be saved. Two React hooks are provided to assist: `settingsManager.onProcessSettingsTabActivation` and `settingsManager.onProcessSettingsContainerClose`.

```

Comment:
Is `SettingEntry` a class or does this refer to the `SettingTabEntry`? (sorry just reading docs, didn't review code...)

Reply:
I have renamed that thing 4 times but SettingEntry should be SettingsTabEntry.

Reply:
Sounds good.

Reply:
updated 'SettingEntry' to 'SettingsTabEntry'.